backend/src/apiserver/main.go

```diff
@@ -92,15 +92,19 @@ func startRpcServer(resourceManager *resource.ResourceManager) {
 	api.RegisterPipelineServiceServer(s, server.NewPipelineServer(resourceManager))
 	api.RegisterExperimentServiceServer(s, server.NewExperimentServer(resourceManager))
 	api.RegisterRunServiceServer(s, server.NewRunServer(resourceManager))
-	api.RegisterJobServiceServer(s, server.NewJobServer(resourceManager))
+	// Temporary disable the job and visualization service in multi-user mode.
```

Comment:
ah this is fine way to disable i guess just the error message might not be as nice. I was originally thinking return not implemented exception in the job and visualization handler

Reply:
Done.